static/app/views/performance/transactionSummary/latencyChart.tsx

```diff
@@ -204,7 +209,16 @@ class LatencyChart extends React.Component<Props, State> {
       location
     );
 
-    const min = parseInt(decodeScalar(location.query.startDuration, '0'), 10);
+    let min: number | undefined = undefined;
+    let max: number | undefined = undefined;
+
+    if (ZOOM_START in location.query) {
+      min = Math.abs(parseInt(decodeScalar(location.query[ZOOM_START], '0'), 10));
+    }
+
+    if (ZOOM_END in location.query) {
+      max = Math.abs(parseInt(decodeScalar(location.query[ZOOM_END], '0'), 10));
```

Comment:
Is defaulting to 0 here correct? Shouldn't it throw out the max value if it's bad and let the backend decide?

Reply:
I've adjusted it.